torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -527,7 +530,7 @@ c10::intrusive_ptr<JitFuture> RequestCallbackNoPython::processRpc(
       return processForwardAutogradReq(rpc, std::move(streams));
     }
     case MessageType::BACKWARD_AUTOGRAD_REQ: {
-      return processBackwardAutogradReq(rpc);
+      return processBackwardAutogradReq(rpc, std::move(streams));
```

Comment:
do we also need to do this for RRef backward?

Reply:
The RRef backward request doesn't have any tensors and eventually calls distributed backward which goes through this codepath.